[2023-10-17T11:19:51.167+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [queued]>
[2023-10-17T11:19:51.174+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [queued]>
[2023-10-17T11:19:51.174+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:51.183+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-06-22 00:00:00+00:00
[2023-10-17T11:19:51.187+0000] {standard_task_runner.py:57} INFO - Started process 785 to run task
[2023-10-17T11:19:51.190+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-06-22T00:00:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp_5d1xmml']
[2023-10-17T11:19:51.190+0000] {standard_task_runner.py:85} INFO - Job 177: Subtask waiting_for_data
[2023-10-17T11:19:51.236+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:51.306+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-06-22T00:00:00+00:00'
[2023-10-17T11:19:51.313+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:51.313+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:51.313+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:51.314+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:51.319+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210622T000000, start_date=20231017T111951, end_date=20231017T111951
[2023-10-17T11:19:51.362+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:51.389+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:37.363+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [queued]>
[2023-10-18T10:43:37.372+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [queued]>
[2023-10-18T10:43:37.373+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:37.389+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-06-22 00:00:00+00:00
[2023-10-18T10:43:37.397+0000] {standard_task_runner.py:57} INFO - Started process 1750 to run task
[2023-10-18T10:43:37.400+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-06-22T00:00:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp5_91rd65']
[2023-10-18T10:43:37.401+0000] {standard_task_runner.py:85} INFO - Job 176: Subtask waiting_for_data
[2023-10-18T10:43:37.449+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:37.516+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-06-22T00:00:00+00:00'
[2023-10-18T10:43:37.530+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:37.534+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210622T000000, start_date=20231018T104337, end_date=20231018T104337
[2023-10-18T10:43:37.549+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 176 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1750)
[2023-10-18T10:43:37.575+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:37.608+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:43:58.463+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [queued]>
[2023-10-18T11:43:58.476+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [queued]>
[2023-10-18T11:43:58.477+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:43:58.489+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-06-22 00:00:00+00:00
[2023-10-18T11:43:58.494+0000] {standard_task_runner.py:57} INFO - Started process 1153 to run task
[2023-10-18T11:43:58.496+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-06-22T00:00:00+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp03g1zxar']
[2023-10-18T11:43:58.497+0000] {standard_task_runner.py:85} INFO - Job 306: Subtask waiting_for_data
[2023-10-18T11:43:58.545+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-06-22T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:43:58.633+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-06-22T00:00:00+00:00'
[2023-10-18T11:43:58.644+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:43:58.644+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:43:58.645+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:43:58.645+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:43:58.655+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210622T000000, start_date=20231018T114358, end_date=20231018T114358
[2023-10-18T11:43:58.709+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:43:58.809+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
